dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsAPIImplTest.java

```diff
@@ -329,6 +329,40 @@ public void Test_Non_Admin_User_With_Portlet_Read_Attempt() throws DotDataExcept
         api.saveSecrets(builder.build(), APILocator.systemHost() , nonAdminUserWithAccessToPortlet);
     }
 
+    /*
```

Comment:
If this is not needed anymore can be deleted.

Reply:
You're right. But I'm gonna need it in one of my next tasks.  that's why I commented it out but still left the code there.